cmd/mungedocs/preformatted.go

```diff
@@ -45,7 +45,8 @@ func updatePreformatted(filePath string, mlines mungeLines) (mungeLines, error)
 // If the file ends on a preformatted line, there must have been an imbalance.
 func checkPreformatBalance(filePath string, mlines mungeLines) (mungeLines, error) {
 	if len(mlines) > 0 && mlines[len(mlines)-1].preformatted {
-		return nil, fmt.Errorf("file ends in preformatted block")
+		//		return mlines, fmt.Errorf("file ends in preformatted block: %s", mlines[len(mlines)-1].data)
+		return mlines, fmt.Errorf("file ends in preformatted block: %s", mlines[len(mlines)-1].data)
```

Comment:
I updated the error message to be more understandable.  I also added another change to avoid printing the entire file when there is an error affecting the whole file.